source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -0,0 +1,198 @@
+====================
+MongoClient Behavior
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+Overview
+--------
+
+In this guide, you can learn about the different settings to control
+the behavior of your ``MongoClient``.
+
+You can control the behavior of your ``MongoCleint`` by creating and passing in a 
+`MongoClientSettings <{+api+}/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.html>`__
+object to the `MongoClients.create() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoClients.html#create(com.mongodb.MongoClientSettings)>`__
+method.
+
+To create a ``MongoClientSettings`` object, use the
+``MongoClientSettings.builder()`` method and chain methods to specify
+your desired settings. After chaining all the settings you want, use the
+``build()`` method to create the ``MongoClientSettings`` object.
+
+Example
+~~~~~~~
+
+This example demonstrates specifying a ``ConnectionString``:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/mcs.java
+   :start-after: begin ConnectionString
+   :end-before: end ConnectionString
+   :language: java
+   :emphasize-lines: 3
+   :dedent:
+
+.. tip::
+
+   If you omit the ``applyConnectionString()`` method, the driver
```

Comment:
We should specify somewhere that order matters: last one wins, i.e. that if you call applyConnectionString after the other apply method, the connection string will take precedence.  If you call it before, whatever you call after will take precedence.